corehq/apps/style/templates/style/includes/modal_report_issue.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-{% if user.is_authenticated %}
+{% if request.user.is_authenticated %}
```

Comment:
oh, that's nasty. it might be clearer to include that check surrounding the includes